store/src/main/java/org/apache/rocketmq/store/ha/autoswitch/AutoSwitchHAService.java

```diff
@@ -432,7 +434,7 @@ private long computeConfirmOffset() {
         long confirmOffset = this.defaultMessageStore.getMaxPhyOffset();
```

Comment:
refactor confirmOffset to newConfirmOffset

Reply:
done, it has been renamed to newConfirmOffset.